frontend/src/components/index.css

```diff
@@ -0,0 +1,145 @@
+header {
+    font-family: Arial, Helvetica, sans-serif
+}
+
+.navbar {
+    display: block;
+    display: flex;
+    font-size: 1.17em;
+    font-family: Arial, Helvetica, sans-serif;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+    font-weight: bold;
+    align-items: right;
+}
+
+.header-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: gray;
+    font-family: Arial, Helvetica, sans-serif;
+
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+}
+
+.links {
+    flex-direction: row;
+    justify-content: right;
+    align-items: center;
+}
+
+.welcome-hook {
+    display: flex;
+    font-weight: bold;
+    font-size: 2em;
+    justify-content: center;
+    
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.welcome-text {
+    display: flex;
+    font-size: 1em;
+    justify-content: center;
+    
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+header a, h3 a {
+    color: var(gray);
+    margin: 0px 14px;
+}
+
+.flex-container {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-around;
+    margin: auto 5%;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.recipe-box {
+    display: flex;
+    flex: 1 1 0px;
+    margin: 20px;
+    flex-flow: column nowrap;
+    align-items: center;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.recipe-header {
+    display: flex;
+    flex: 1 1 0px;
+    flex-flow: row nowrap;
+    align-items: left;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.descriptions {
+    display: flex;
+    margin: 5%;
+    flex-direction: row;
+    text-align: center;
+    /*background-color: red;*/
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.content-box {
+    display: flex;
+    flex-direction: row;
+}
+
+
+.food-image {
+    width: 200px;
+}
+
+.recipe-header {
+    display: flex;
+}
+
+
+.about-container {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    justify-content: center;
+    margin: 25px 0px;
+}
+
+.about-title {
+    display: flex;
+    font-family: Arial, Helvetica, sans-serif;
+    font-weight: bold;
+    justify-content: center;
+    font-size: 30px;
+}
+
+.about-text {
+    display: flex;
```

Comment:
I was just going through your css very quickly, it looks like you are using display: flex a lot. You should only need to use it in some of your outermost tags. This is a great resource for improving your skills when using flex! https://flexboxfroggy.com/